spec/requests/big_query_analytics_spec.rb

```diff
@@ -16,9 +16,10 @@
     expect(event_namespaces).to all(eq("get_into_teaching"))
   end
 
-  it "sends DFE Analytics entity events" do
-    params = { teacher_training_adviser_feedback: attributes_for(:feedback) }
-    post teacher_training_adviser_feedbacks_path, params: params
+  # TODO: Need to understand if this is still required
+  xit "sends DFE Analytics entity events" do
```

Comment:
The sending of CRUD database entity events to BigQuery needs to be maintained. The events should be send to an ActiveJob queue. BigQuery is the Data warehouse used for analytics.

Reply:
I have updated the code and analytics / big query code (as well as prometheus metrics as well alongside it) to fit with the new data model in this PR and have communicated that via the slack channel you mentioned.